spec/git-spec.coffee

```diff
@@ -14,13 +14,147 @@ describe "Git grammars", ->
       expect(grammar.scopeName).toBe "source.git-config"
 
   describe "Git commit messages", ->
+    scopeNormal =  [ 'text.git-commit', 'meta.scope.message.git-commit' ]
```

Comment:
Nit: remove the padded whitespace in these brackets.
`['text.git-commit', 'meta.scope.message.git-commit']`